pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -1482,6 +1495,7 @@ func admitVMICreate(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
 
 	causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("spec"), &vmi.Spec)
 	causes = append(causes, ValidateVirtualMachineInstanceMandatoryFields(k8sfield.NewPath("spec"), &vmi.Spec)...)
+	causes = append(causes, ValidateVirtualMachineInstanceMetadata(k8sfield.NewPath("spec"), vmi)...)
```

Comment:
This also needs to be validated at the VM template level. so we'll need this added somewhere in admitVMCreate